rest-api/src/main/java/gov/cms/qpp/conversion/api/security/JwtAuthorizationFilter.java

```diff
@@ -79,6 +80,7 @@ protected void doFilterInternal(HttpServletRequest request,
 	 * @param tokenHeader Object holding the token
 	 * @return data map of the token parsed
 	 */
+	@SuppressWarnings("unchecked")
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Suppressing warnings is not allowed [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1309)

Reply:
Big brother requires acquiescence

Reply:
> I am disinclined to acquiesce to his request.